pkg/controller/admissionchecks/provisioning/controller_test.go

```diff
@@ -87,6 +89,30 @@ func requestWithCondition(r *autoscaling.ProvisioningRequest, conditionType stri
 	return r
 }
 
+func provReqConfigWithRetryLimit(prc *kueue.ProvisioningRequestConfig, limit int32) *kueue.ProvisioningRequestConfig {
```

Comment:
I think it would be better to add a builder for testing of ProvisioningRequestConfig. but I'm ok to leave it as a follow up.

Reply:
Thanks, I'll do it in a follow up